client/extensions/wp-super-cache/preload-tab.jsx

```diff
@@ -0,0 +1,184 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormToggle from 'components/forms/form-toggle/compact';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import WrapSettingsForm from './wrap-settings-form';
+
+/**
+ * Preload Tab
+ * @returns {Jsx} the template
+ */
+const PreloadTab = ( {
+	fields: {
+		minimum_preload_interval,
+		wp_cache_preload_interval,
+		wp_cache_preload_posts,
+		wp_cache_preload_on,
+		wp_cache_preload_taxonomies,
+		wp_cache_preload_email_volume,
+	},
+	translate,
+	handleToggle,
+	handleChange,
+	handleSelect,
+} ) => {
+	const preloadPostsSelectValues = [ 17, 34, 51, 68, 85, 102, 119 ];
+	const statusEmailAmountSelectValues = [
+		{ value: 'none', description: translate( 'No emails' ) },
+		{ value: 'many', description: translate( 'Many emails, 2 emails per 100 posts.' ) },
+		{ value: 'medium', description: translate( 'Medium, 1 email per 100 posts.' ) },
+		{ value: 'less', description: translate( 'Less emails, 1 at the start and 1 at the end of preloading all posts.' ) }
```

Comment:
I now understand this is a port of the original plugin, but I think this should read _Fewer emails_. Can I get a quick-quick sanity check from the all-knowing @benhuberman? :)